addons/sale/views/product_views.xml

```diff
@@ -51,7 +51,7 @@
                     attrs="{'invisible': [('sale_ok', '=', False)]}">
                     <div class="o_field_widget o_stat_info">
                         <span class="o_stat_value">
-                            <field name="sales_count" widget="statinfo" nolabel="1" class="mr4"/>
+                            <field name="sales_count" widget="statinfo" nolabel="1"/>
```

Comment:
There are a lot of occurrences of this problem in the code base. Is there no way to use parseFieldNode to communicate
to the concrete field the original classList and use it? Is it a good idea?

Reply:
(the problem occurs with things other than mr4)